verification/curator-service/ui/src/components/util/date.test.tsx

```diff
@@ -0,0 +1,9 @@
+import renderDate from './date';
+
+describe('Dates', () => {
+    it('are padded', () => {
+        expect(renderDate(new Date(Date.UTC(2020, 6, 9)))).toEqual(
+            '2020-07-09',
```

Comment:
I just can't believe `Date.UTC` uses a zero-based month.

Reply:
well, at least it's consistently bad.